test/contact-forms.spec.js

```diff
@@ -119,17 +119,26 @@ describe('contact forms', () => {
   });
 
   it('#59 - msf-goma create person', async () => {
-    const result = await harness.fillContactForm('goma-person', [
+    const result = await harness.fillContactCreateForm('goma-person', [
       'PARENT', 'patient', '123', 'Full Name', '1990-10-08', 'male',
       '555-123-4567', 'false', 'eng', 'yes', 'second', 'no', 'unknown',
       ['diabetes'], '', 'notes'
     ]);
+    expect(result.errors).to.be.empty;
+  });
 
+  it('create and edit household contact', async () => {
+    let result = await harness.fillContactCreateForm('household_contact', ...[['Head', 'female', 'over5', 'no', '20', '', '', 'sister']]);
+    expect(result.errors).to.be.empty; // create the person
+    const contactBeforeEdit = { ...result.contacts[0] };
+    harness.subject = result.contacts[0]; // set the person as subject
+    result = await harness.fillContactEditForm('household_contact', []);
     expect(result.errors).to.be.empty;
+    Object.keys(harness.subject).forEach(key => contactBeforeEdit[key] && expect(harness.subject[key], key).to.be.eq(contactBeforeEdit[key]));
```

Comment:
i still don't love this. what about chai exclude `expect({ a: 'a', b: 'b' }).excluding('a').to.deep.equal({ b: 'b' })`? https://www.npmjs.com/package/chai-exclude